jerry-core/api/jerry.c

```diff
@@ -3668,6 +3751,67 @@ jerry_foreach_object_property (const jerry_value_t obj_val, /**< object value */
   return false;
 } /* jerry_foreach_object_property */
 
+/**
+ * FromPropertyDescriptor abstract operation.
+ *
+ * @return new jerry_value_t - if success
+ *         value marked with error flag - otherwise
+ */
+jerry_value_t
+jerry_from_property_descriptor (const jerry_property_descriptor_t *src_prop_desc_p) /**< property descriptor */
+{
+  jerry_assert_api_available ();
+
+  ecma_property_descriptor_t prop_desc = jerry_property_descriptor_to_ecma (src_prop_desc_p);
+
+  if (ECMA_IS_VALUE_ERROR (prop_desc.value))
+  {
+    jerry_throw (prop_desc.value);
+  }
+
+  ecma_object_t *desc_obj_p = ecma_op_from_property_descriptor (&prop_desc);
+
+  return ecma_make_object_value (desc_obj_p);
+} /* jerry_from_property_descriptor */
+
+/**
+ * ToPropertyDescriptor abstract operation.
+ *
+ * @return true - if success
+ *         false - if error occurs
+ */
+bool
+jerry_to_property_descriptor (jerry_value_t obj_value, /**< object value */
```

Comment:
Please change the return value of this function to `jerry_value_t` which returns:
 - thrown error - if any conversion error happens
 - ECMA_VALUE_TRUE - otherwise.